common/src/main/java/gov/cms/ab2d/common/repository/JobRepository.java

```diff
@@ -19,6 +19,7 @@
 public interface JobRepository extends JpaRepository<Job, Long> {
 
     @Modifying
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Curious what prompted this change

Reply:
I got some weird test failures that appeared to be DB related and this helped. Originally, I had logging in bfd since that was were the query happened but then I realized I'd have to pass a bunch of data to the BFD client which didn't really belong there and it would need to be able to use the database in that which caused its own issues. I moved it up a layer that already had access to all the information (job, user, etc) but since this didn't seem to hurt and may even help and the other modifying method uses it, I decided to leave it.